docs/node-operators/resources/faq.md

```diff
@@ -2,71 +2,85 @@
 description: List of frequently asked questions.
 ---
 
-import multisig from './assets/multisig.png';
 
 # Node operator FAQ
+
+import multisig from './assets/multisig.png';
+
 Here's a list of frequently asked questions.
 
-### 1. How do I know if my validator node is working? {#1}
+## 1. How do I know if my validator node is working? {#1}
```

Comment:
Either change this to level-3 heading for consistency with others, or change all headings (except the first level-1 heading) to level-2 headings. The formatting is inconsistent now, I strongly recommend that you run the project locally to make sure that everything is rendered as expected.